config/crd/bases/keda.sh_clustertriggerauthentications.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.3.0
+    controller-gen.kubebuilder.io/version: v0.5.0
```

Comment:
I think the best thing would be to omit this change from the PR.

Reply:
Done